Services/Masterdata/Services.Masterdata/Extensions/MasterdataTypeExtensions.cs

```diff
@@ -50,6 +52,19 @@ public static void Sanitize(this MasterdataCreateModel model, IHtmlSanitizer htm
                 model.Metadata = JsonSerializer.Deserialize<JsonElement>(metadataString);
             }
         }
+        if (model.Tags != null)
```

Comment:
This class contains lots of duplicate code (sanitization and tag processing). You should take the DRY principle into account (Dont Repeat Yourself) and create private methods, so you've to write the logic only once = one place in the code that can contain bugs/ needs to be maintained instead of N places

Reply:
You are right. I'm familiar with the DRY principle, but unfortunately yesterday I wasn't able to move the duplicated code in a private method. Today the solution seemed pretty easy!